src/http/request.ts

```diff
@@ -17,14 +17,39 @@ const httpError = {
   },
 }
 
-function printWarnings() {
+function printWarnings(config: {ignoreWarnings?: string | RegExp | Array<string | RegExp>} = {}) {
   const seen: Record<string, boolean> = {}
+
+  // Helper function to check if a warning should be ignored
+  const shouldIgnoreWarning = (message: string): boolean => {
+    if (!config.ignoreWarnings) return false
```

Comment:
This means that `ignoreWarnings: ''` end up not ignoring anything while `ignoreWarnings: ['']` ignores everything. A bit of an edge case, but would be easily fixed by `config.ignoreWarnings === undefined`)

Reply:
Good catch! Fixed.